examples/config_files/settings/settings_user.rst

```diff
@@ -1,6 +1,305 @@
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=apple-clang
+    compiler.cppstd=gnu98
+    compiler.libcxx=libc++
+    compiler.version=12.0
+    os=Macos
+
+    Profile build:
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=apple-clang
+    compiler.cppstd=gnu98
+    compiler.libcxx=libc++
+    compiler.version=12.0
+    os=Macos
+    ...
+    -------- Installing (downloading, building) binaries... --------
+    pkg/1.0: Copying sources to build folder
+    pkg/1.0: Building your package in /Users/franchuti/.conan2/p/t/pkg705693c0039ff/b
+    pkg/1.0: Aggregating env generators
+    pkg/1.0: Package '20496b332552131b67fb99bf425f95f64d0d0818' built
+    pkg/1.0: Build folder /Users/franchuti/.conan2/p/t/pkg705693c0039ff/b
+    pkg/1.0: Generated conaninfo.txt
+    pkg/1.0: Generating the package
+    pkg/1.0: Temporary package folder /Users/franchuti/.conan2/p/t/pkg705693c0039ff/p
+    pkg/1.0 package(): WARN: No files in this package!
+    pkg/1.0: Package '20496b332552131b67fb99bf425f95f64d0d0818' created
+    pkg/1.0: Created package revision df76396a179b74f9e248149c128b0fd6
+    pkg/1.0: Full package reference: pkg/1.0#637fc1c7080faaa7e2cdccde1bcde118:20496b332552131b67fb99bf425f95f64d0d0818#df76396a179b74f9e248149c128b0fd6
+    pkg/1.0: Package folder /Users/franchuti/.conan2/p/pkg9be0d15c9cd72/p
+
+
+.. code-block:: bash
+    :caption: **Using the new OS and its sub-setting**
+    :emphasize-lines: 11,12,34
+
+    $ conan create . -s os=webOS -s os.sdk_version=7.0.0
+    ...
+    Profile host:
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=apple-clang
+    compiler.cppstd=gnu98
+    compiler.libcxx=libc++
+    compiler.version=12.0
+    os=webOS
+    os.sdk_version=7.0.0
+
+    Profile build:
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=apple-clang
+    compiler.cppstd=gnu98
+    compiler.libcxx=libc++
+    compiler.version=12.0
+    os=Macos
+    ...
+    -------- Installing (downloading, building) binaries... --------
+    pkg/1.0: Copying sources to build folder
+    pkg/1.0: Building your package in /Users/franchuti/.conan2/p/t/pkg929d53a5f06b1/b
+    pkg/1.0: Aggregating env generators
+    pkg/1.0: Package 'a0d37d10fdb83a0414d7f4a1fb73da2c210211c6' built
+    pkg/1.0: Build folder /Users/franchuti/.conan2/p/t/pkg929d53a5f06b1/b
+    pkg/1.0: Generated conaninfo.txt
+    pkg/1.0: Generating the package
+    pkg/1.0: Temporary package folder /Users/franchuti/.conan2/p/t/pkg929d53a5f06b1/p
+    pkg/1.0 package(): WARN: No files in this package!
+    pkg/1.0: Package 'a0d37d10fdb83a0414d7f4a1fb73da2c210211c6' created
+    pkg/1.0: Created package revision 6a947a7b5669d6fde1a35ce5ff987fc6
+    pkg/1.0: Full package reference: pkg/1.0#637fc1c7080faaa7e2cdccde1bcde118:a0d37d10fdb83a0414d7f4a1fb73da2c210211c6#6a947a7b5669d6fde1a35ce5ff987fc6
+    pkg/1.0: Package folder /Users/franchuti/.conan2/p/pkgb3950b1043542/p
+
+.. code-block:: bash
```

Comment:
I think we should add a short introduction before each code block to explain what we are doing there and the expected results

Reply:
I think that the `caption` could be enough because we're only creating with different settings. If you think so I can remove it and add a sentence instead

Reply:
Sorry I did not see the caption, good enough 👍